### PR TITLE
@import must occur at top of stylesheet

### PR DESCRIPTION
**Pull request by mikepierce**
Specified that @import rules have to come before any declarations in the stylesheet, as per https://www.w3.org/TR/CSS1/

### Description

Added stipulation that @import must occur at the top of a stylesheet

### Motivation

So readers know they can't @import at the bottom of their stylesheet

### CODE REVIEW

File: files/en-us/web/css/@import/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ browser-compat: css.at-rules.import
 
 {{CSSRef}}
 
-The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other stylesheets.
+The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other stylesheets. An `@import` rule _must_ occur at the top of stylesheet before any declarations, otherwise it will be ignored.
```
Comment (dipikabh) [inline files/en-us/web/css/@import/index.md:16]:
```suggestion
The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other valid stylesheets. An `@import` rule _must_ be defined at the top of stylesheet, before any other at-rule and style declaration, else it will be ignored.
```

(Referring https://drafts.csswg.org/css-cascade-5/#at-import)

Although the [Description](https://developer.mozilla.org/en-US/docs/Web/CSS/@import#description) section mentions the following along with the appropriate qualifiers, I think it is nice to emphasize it in the intro para:
"Imported rules must come before all other types of rules, except [@charset](https://developer.mozilla.org/en-US/docs/Web/CSS/@charset) rules and layer creating [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) statements."

**Comment by estelle** [comment]
`\@import` can be preceded by `\@charset`  and `\@layers`

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @mikepierce, welcome to the MDN Web Docs project and community! 🎉 Thanks for opening this pull request to improve the content. 🙌 

Please see my edit suggestions to your changes.

File: files/en-us/web/css/@import/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/web/css/@import/index.md:16]: As Estelle has pointed out, maybe change the phrase "before any other at-rule and style declaration" to "before any other at-rule (except [@charset](https://developer.mozilla.org/en-US/docs/Web/CSS/@charset) and [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer)) and style declaration"

File: files/en-us/web/css/@import/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ browser-compat: css.at-rules.import
 
 {{CSSRef}}
 
-The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other stylesheets.
+The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other valid stylesheets. An `@import` rule _must_ be defined at the top of stylesheet, before any other at-rule and style declaration, else it will be ignored.
```
Comment (dipikabh) [inline files/en-us/web/css/@import/index.md:16]:
```suggestion
The **`@import`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At-rule) is used to import style rules from other valid stylesheets. An `@import` rule _must_ be defined at the top of stylesheet, before any other at-rule (except [@charset](/en-US/docs/Web/CSS/@charset) and [@layer](/en-US/docs/Web/CSS/@layer)) and style declaration, else it will be ignored.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Tiny nit as mentioned in previous review

**Review by dipikabh — APPROVED** [review]
Thank you and congratulations on landing your first changes on MDN! 🎉

**Pull request merged**